aikau/src/main/resources/alfresco/services/SiteService.js

```diff
@@ -111,6 +142,20 @@ define(["dojo/_base/declare",
                { label: "create-site.dialog.type.collaboration", value: "site-dashboard" }
             ];
          }
+
+         if (this.additionalSitePresets && this.additionalSitePresets.length)
```

Comment:
Well, this does a couple of things... firstly it is a crude sniff test to check that it *might* be an array, and if it is an array then using falsy logic it only proceeds with the concat operation if there actually is something concatenate.